ddtrace/contrib/flask/patch.py

```diff
@@ -325,8 +331,10 @@ def traced_wsgi_app(pin, wrapped, instance, args, kwargs):
             config.flask,
             method=request.method,
             url=request.base_url,
-            query=request.query_string,
+            raw_uri=request.url,
+            query=query,
             request_headers=request.headers,
+            request_path_params=request.args,
```

Comment:
Based on the docstring I think this is wrong? and we do not have access to the route here, only at the end of the request for Flask.

Reply:
That one's on me - I mixed two addresse, let me update the docstring